### ISSUE TITLE
"Search for a space" from My Spaces can displace scrollbar

### ISSUE THREAD

**Issue by andrewazores** [openshiftio/openshift.io#4269]
##### Issue Overview

When appropriately sized, the My Spaces page will display a standard scroll bar. If the "Search for a space" toolbar button is clicked and the modal dialog appears, this scroll bar may disappear, causing the background (My Spaces view) to shift slightly. When the modal is dismissed the scroll bar may reappear, which shifts My Spaces back. This behaviour has been reproduced in Chromium/Chrome but not Firefox.

##### Expected Behaviour

The scroll bar should either always show or always hide, regardless of modal show/hide state.

##### Current Behaviour

The scroll bar may hide when the modal is shown and vice-versa.

##### Steps To Reproduce

1. Open Chrome/Chromium browser
2. Navigate to My Spaces (top left menu > My Spaces)
3. Resize browser window so scroll bar appears
4. Click "Search for a space" toolbar button. If it does not appear, ensure Feature Opt-In in User Settings is set to Experimental, and retry this step.
5. Observe scroll bar becomes hidden
6. Dismiss dialog
7. Observe scroll bar becomes shown

**Comment by aptmac** [comment]
FWIW, this is intentional on the Bootstrap side of things [[0]](https://github.com/twbs/bootstrap/blob/283ab30164f0f58ffb13063d800b7a2ee686bb8a/scss/_modal.scss#L7).

Having said that, I played around a little bit with this, and if the `overflow` value is set to `scroll` then the scrollbar will persist. However, the body gets appended with a `modal-open` attribute and  `padding-right: 15px` styling.
![open](https://user-images.githubusercontent.com/10425301/45311459-28d88380-b4f7-11e8-9b8b-efa26ddde486.png)
(note the gap on the pf-toolbar)
![gap](https://user-images.githubusercontent.com/10425301/45311508-4e658d00-b4f7-11e8-9af5-16159c8b71cf.png)
It's also interesting to note that when the modal hides, it resets this value, but leaves a `padding-right: 0px` on the body.
![closed](https://user-images.githubusercontent.com/10425301/45311470-2fff9180-b4f7-11e8-8e3e-dd86e9d4510d.png)

**Comment by joshuawilson** [comment]
Is this still an issue after the changes you made?

**Comment by andrewazores** [comment]
Yes, although after Alex's comment above this may be a WONTFIX deal if it's an intentional Bootstrap quirk/feature.
